packages/stark-ui/src/modules/table/components/column.component.html

```diff
@@ -0,0 +1,37 @@
+<ng-container matColumnDef>
+	<!-- custom header that supports multi-column sorting -->
+	<th mat-header-cell *matHeaderCellDef [ngClass]="{'sortable': sortable, 'filtering': filterValue}">
+		<div class="header-cell-content">
+			<div [ngSwitch]="sortDirection" class="sort-header" (click)="onSortChange()">
+				<span>{{ headerLabel || name }}</span>
```

Comment:
Not for now, translation is not part of this PR, as I mentioned in the description, this is only about nested data, sorting and filtering, otherwise this PR will be huge.

Translation and other features will be part of another PR 😉

Reply:
Ok. That means we'll have to be really careful to not forget anything in the next PR which will contain translations